astropy/timeseries/periodograms/lombscargle/implementations/scipy_impl.py

```diff
@@ -23,6 +28,11 @@ def lombscargle_scipy(t, y, frequency, normalization="standard", center_data=Tru
     center_data : bool, optional
         if True, pre-center the data by subtracting the weighted mean
         of the input data.
+    fit_mean : bool (default: False)
```

Comment:
I think this should default to True like in other implementations. Since this is a non-public API, there is no problem with backward compatibility; we just need to make sure that the public APIs call this function in a backward-compatible way.